Framework/PythonInterface/test/python/plugins/algorithms/IndirectCalibrationTest.py

```diff
@@ -16,6 +16,17 @@ def test_simple(self):
         self.assertEqual(cal_ws.getNumberHistograms(), 51)
         self.assertEqual(cal_ws.blocksize(), 1)
 
+    
+    def test_logs(self):
+        cal_ws = IndirectCalibration(InputFiles='IRS38633.raw',
+                                            DetectorRange=[3,53],
+                                            PeakRange=[62000,65000],
+                                            BackgroundRange=[59000,61000],
+                                            LoadLogFiles=True)
+
+        self.assertEqual(cal_ws.getNumberHistograms(), 51)
+        self.assertEqual(cal_ws.blocksize(), 1)
+        self.assertEqual(cal_ws.run().getProperty('SampleLogs'))
```

Comment:
Sorry I think I might not have explained this very well!
`cal_ws.run()` returns you a handle to the sample logs. In order to look at a specific value in the logs e.g. `current_period` you can do:
`cal_ws.run().getProperty('current_period')`
from there, we can then use the `assertEqual(actual, expected)` to check that the value of the current_period is what we expect it to be